apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
Before doing that you might want to test and see if `NODE_ENV` is already set to `"production"` in that build (static site vs the live reload site we normally do for local development) - it might be just what we need.